src/Leela.cpp

```diff
@@ -238,6 +243,15 @@ static void parse_commandline(int argc, char *argv[]) {
             exit(EXIT_FAILURE);
         }
     }
+    if (cfg_precision == precision_t::AUTO) {
+        // Auto precision is not supported for tune only cases.
+        // Since full tuner implies tune only, check that too.  #1973
+        if (cfg_sgemm_exhaustive || cfg_tune_only) {
```

Comment:
Is there a good reason for repeating the full tuner implying tune only handling? I’d prefer if the decision is made only in one place.

Reply:
Yes I think it makes sense.  I'll push achange.